src/app/components/content-action-bar/ContentActionBar.jsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+
+const buttonStyle = {
```

Comment:
I don't like those constants here. classNames should be passed as props if there will be different variants of the component

Reply:
Yer I agree these can now be removed. Initially, there were a few classes being applied to various elements when I initially added them in, then optimised it a bit during further development but I didn't clock that this is actually redundant now. Good spot.